resources/web/wwi/viewer.js

```diff
@@ -1193,7 +1202,8 @@ function renderGraphs() {
       // set min-width to be 2/3 of the max-width otherwise the text might become too small
       const element = document.querySelector('#' + id);
       const style = element.getAttribute('style');
-      element.setAttribute('style', style + ' min-width:' + Math.floor(0.66 * parseInt(style.split('max-width:')[1].split('px'))) + 'px;');
+      element.setAttribute('style', style + ' min-width:' + Math.floor(0.66 * parseInt(style.split('max-width:')[1]
+        .split('px'))) + 'px;');
```

Comment:
If we follow the best practice this should be split at a higher level:
```suggestion
      element.setAttribute('style',
        style + ' min-width:' + Math.floor(0.66 * parseInt(style.split('max-width:')[1].split('px'))) + 'px;');
```